Documentation.md

```diff
@@ -906,128 +941,95 @@ currently this is limited to softphone-enabled voice contacts.
 
 ### `connection.hold()`
-```
+```js
 conn.hold({
-   success: function() { ... },
-   failure: function() { ... }
+   success: function() { /* ... */ },
+   failure: function(err) { /* ... */ }
 });
 ```
 Put this connection on hold.
 
-Optional success and failure callbacks can be provided to determine whether the operation was successful.
+Optional success and failure callbacks can be provided to determine if the operation was successful.
 
 ### `connection.resume()`
-```
+```js
 conn.resume({
-   success: function() { ... },
-   failure: function() { ... }
+   success: function() { /* ... */ },
+   failure: function(err) { /* ... */ }
 });
 ```
 Resume this connection if it was on hold.
 
-Optional success and failure callbacks can be provided to determine whether the operation was successful.
+Optional success and failure callbacks can be provided to determine if the operation was successful.
 
 ## VoiceConnection API
 The VoiceConnection API provides action methods (no event subscriptions) which can be called to manipulate the state
-of a particular voice connection within a contact.  Like contacts, connections come and go. It is good practice not
+of a particular voice connection within a contact. Like contacts, connections come and go. It is good practice not
 to persist these object or keep them as internal state. If you need to, store the `contactId` and `connectionId`
 of the connection and make sure that the contact and connection still exist by fetching them in order from
 the `Agent` API object before calling methods on them.
 
-### `voiceConnection.sendDigits()`
-```
-conn.sendDigits(digits, {
-   success: function() { ... },
-   failure: function() { ... }
-});
-```
-Send a digit or string of digits through this connection.
-
-### `voiceConnection.hold()`
-```
-conn.hold({
-   success: function() { ... },
-   failure: function() { ... }
-});
-```
-Put this voice connection on hold.
-
-### `voiceConnection.resume()`
-```
-conn.resume({
-   success: function() { ... },
-   failure: function() { ... }
-});
-```
-Only has an effect if this connection was on hold when called.
-
-### `voiceConnection.isOnHold()`
-```
-if (conn.isOnHold()) { ... }
-```
-Returns true if this connection is on hold, false otherwise.
-
 ### `voiceConnection.getMediaInfo()`
-```
-const mediaInfo = conn.getMediaInfo();
+```js
+var mediaInfo = conn.getMediaInfo();
 ```
 Returns the media info object associated with this connection.
 
 ### `voiceConnection.getMediaType()`
+```js
+if (conn.getMediaType() === "softphone") { ... }
 ```
-if (conn.getMediaType()==="SOFTPHONE") { ... }
-```
-Returns the softphone media type enum: `"SOFTPHONE"`.
+Returns the `MediaType` enum value: `"softphone"`.
 
 ### `voiceConnection.getMediaController`
-```
-const voiceController = conn.getMediaController();
+```js
+var voiceController = conn.getMediaController();
 ```
 Get the media controller associated with this connection.
```

Comment:
After writing the TypeScript for this, I'm wondering whether this should be a public API...